workshops/templates/workshops/person.html

```diff
@@ -100,6 +100,17 @@
     {% else %}
     <h5>No lessons.</h5>
     {% endif %}
+
+    {% if languages %}
+    <h5>Languages:</h5>
+    <ul>
+      {% for l in languages %}
```

Comment:
I was adhering to the naming convention that was followed around this code. Will make a change.